onnxruntime/test/autoep/test_autoep_selection.cc

```diff
@@ -353,6 +356,150 @@ TEST(AutoEpSelection, PreferNpu) {
                        OrtExecutionProviderDevicePolicy::OrtExecutionProviderDevicePolicy_PREFER_NPU);
 }
 
+static OrtStatus* PolicyDelegate(_In_ const OrtEpDevice** ep_devices,
```

Comment:
Maybe these need `ORT_API_CALL` so that the x86 pipeline sees them as matching the signature of `EpSelectionDelegate`